test/acceptance/specs/APIGatekeeperUnblockApplicationSpec.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package acceptance.specs
+
+import acceptance.pages._
+import com.github.tomakehurst.wiremock.client.WireMock._
+
+import scala.io.Source
+
+class APIGatekeeperUnblockApplicationSpec extends APIGatekeeperBaseSpec {
+
+  val appName = "Automated Test Application - Blocked"
+  val unblockedAppName = "Automated Test Application - Unblocked"
+
+  feature("Unblock an application") {
+    scenario("I can unblock an application") {
+      stubBlockedApplication(applicationToUnblock)
+      stubApplicationForUnblockSuccess()
+
+      When("I navigate to the application page")
+      navigateToApplicationPageFor(appName)
+
+      And("I choose to unblock the application")
+      selectToUnblockApplication()
+
+      Then("I am successfully navigated to the Unblock Application Success page")
+      on(UnblockApplicationSuccessPage)
+      assert(UnblockApplicationSuccessPage.bodyText.contains("Application unblocked"))
+    }
+
+    scenario("I cannot unblock an application that is already unblocked") {
+      stubApplication(unblockedApplication)
```

Comment:
You could reuse applicationToBlock or possibly rename applicationToBlock to unblockedApplicaiton